Makefile

```diff
@@ -92,19 +81,78 @@ lint:
 misspell:
 	@output=`misspell -error $(ALL_DOC)` && echo misspell finished successfully || (echo misspell errors:\\n$$output && exit 1)
 
+# --------------------------
+#  CI
+# --------------------------
+
+# Adds license headers to files that are missing it, quiet tests
+# so full output is visible at a glance.
+.PHONY: precommit
+precommit: addlicense lint misspell test_quiet
+
+# Checks for the presence of required license headers, runs verbose
+# tests for complete information in CI job.
+.PHONY: presubmit
+presubmit: checklicense lint misspell test
+
+# --------------------------
+#  Build and Test 
+# --------------------------
+
+GO_BUILD_OUT ?= ./bin/otelopscol
 .PHONY: build
 build:
-	go build -tags=$(GO_TAGS) -o ./bin/$(OTELCOL_BINARY) $(LD_FLAGS) ./cmd/otelopscol
+	go build -tags=$(GO_BUILD_TAGS) -o $(GO_BUILD_OUT) $(LD_FLAGS) ./cmd/otelopscol
+
+OTELCOL_BINARY = google-cloud-metrics-agent_$(GOOS)_$(GOARCH)$(EXTENSION)
+.PHONY: build_full_name
+build_full_name:
+	go build -tags=$(GO_BUILD_TAGS) -o ./bin/$(OTELCOL_BINARY) $(LD_FLAGS) ./cmd/otelopscol
 
 .PHONY: test
 test:
-	go test -tags=$(GO_TAGS) -v -race ./...
+	go test -tags=$(GO_BUILD_TAGS) -v -race ./...
 
-# googet (Windows)
+.PHONY: test_quiet
+test_quiet:
+	go test -tags=$(GO_BUILD_TAGS) -race ./...
```

Comment:
This is essentially the same command as in the `test` target, but without the `-v`. I wonder if it makes sense to define one in terms of another, e.g.:
```
.PHONY: test
test:
	go test -tags=$(GO_BUILD_TAGS) $(VERBOSE_TESTS) -race ./...

.PHONY: test_verbose
test_verbose:
	$(MAKE) VERBOSE_TESTS=-v test
```

Reply:
Done in [a580ef7](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/pull/157/commits/a580ef727c331ca6e3c78d47744f713d0396b907)